<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Метод отрисовки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="method_box">
        <item>
         <property name="text">
          <string>Канонического уравнения</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Параметрического уравнения</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Алгоритма Брезенхема</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Алгоритма средней точки</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Библиотечный</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Цвет линии</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="color_box">
        <item>
         <property name="text">
          <string>Черный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Синий</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Белый</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Фигура</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="figure_box">
        <item>
         <property name="text">
          <string>Окружность</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Эллипс</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Центр</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="x_center">
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="y_center">
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>250</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Полуоси</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="pol_a">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="pol_b">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
                <property name="value">
                 <number>150</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18"/>
      </item>
      <item>
       <widget class="QPushButton" name="drawfigure">
        <property name="text">
         <string>Отрисовать фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Нач. полуоси</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>A / R</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pol_a_spec">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pol_b_spec">
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>25</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Центр спектра</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="x_center_spec">
                <property name="maximum">
                 <number>999</number>
                </property>
                <property name="value">
                 <number>450</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="y_center_spec">
                <property name="maximum">
                 <number>999</number>
                </property>
                <property name="value">
                 <number>300</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Шаг изменения </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="pol_edit_spec">
              <item>
               <property name="text">
                <string>A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>B</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pol_step_spec">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QComboBox" name="type_spec">
              <item>
               <property name="text">
                <string>Кол-во</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Конечн. знач</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pol_max_spec">
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="drawSpecture">
        <property name="text">
         <string>Построить спектр</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="clearCanvas">
        <property name="text">
         <string>Очистить холст</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>456</width>
             <height>559</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="canvas">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>500</width>
                <height>500</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Канвас</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
              <property name="indent">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Определение временных характеристик алгоритмов построения:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Окружностей</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Эллипсов</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
